packages/instant/src/containers/selected_asset_buy_button.ts

```diff
@@ -0,0 +1,55 @@
+import { BuyQuote } from '@0xproject/asset-buyer';
+import * as _ from 'lodash';
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { Dispatch } from 'redux';
+
+import { Action, actions } from '../redux/actions';
+import { State } from '../redux/reducer';
+import { AsyncProcessState } from '../types';
+
+import { BuyButton } from '../components/buy_button';
+
+export interface SelectedAssetBuyButtonProps {}
+
+interface ConnectedState {
+    text: string;
+    buyQuote?: BuyQuote;
```

Comment:
same comment about passing the same quoteInfo here then we are using in OrderDetails

Reply:
I'm not sure what you mean. In terms of the API they could be different, but technically in this implementation they are not.